search-app/src/cmr/search/results_handlers/opendata_results_handler.clj

```diff
@@ -243,18 +255,80 @@
     time
     gran-time))
 
+(defn- title->citation-title
+  "Format citation name."
+  [title version]
+  (when title
```

Comment:
Some alternatives - I'm thinking the one at the bottom might be the most readable?
```
  (when title
    (let [version-string (if version (format ". Version %s" version) "")
      (format "%s%s" title version-string))
```
or
```
  (when title
    (let [version-string (if version (str ". Version " version) "")
      (str title version-string))
```